### PR TITLE
Final JS Examples Pass

### PR DESCRIPTION
**Pull request by maboa**
I've now been through the all JS examples and modified code so that they adhere to the current 'guidelines'. These guidelines had evolved over time.

The `console.log` output has also become more detailed over time so `// expected output` was also brought up to date in line with that.

Note - where examples used the elements introduced post-ES6 implementation, I took the opportunity to use ES6 syntax.

In very few cases I took the opportunity to make the examples better - usually adding to them.

I've also removed some comments where I thought they were not needed - hopefully the examples will explain themselves! :)

Note: a couple of examples produce errors due to what appears to be a bug in the editor, these are detailed here https://github.com/mdn/interactive-examples/issues/356 - I'm pretty sure these worked in earlier versions of the editor, so I'm guessing we introduced the error. Happy to take a look if folk think that would be useful.

### CODE REVIEW

File: live-examples/js-examples/expressions-voidoperator.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <pre>
 <code id="static-js">void function test() {
-  console.log("boo!");
+  console.log('boo!'');
```
Comment (wbamberg) [inline live-examples/js-examples/expressions-voidoperator.html]: -> console.log('boo!')

File: live-examples/js-examples/globalprops-nan.html [hunk 2]
```diff
@@ -6,10 +6,10 @@
   return x;
 }
 
-console.log(sanitise("1"));
+console.log(sanitise('1'));
 // expected output: 1
```
Comment (wbamberg) [inline live-examples/js-examples/globalprops-nan.html]: This actually logs "1"

Comment (wbamberg) [inline live-examples/js-examples/globalprops-nan.html]: This actually logs "1" (this comment is for line 10 but GitHub)

File: live-examples/js-examples/symbol-iterator.html [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <pre>
-<code id="static-js">var iterable1 = new Object();
+<code id="static-js">const iterable1 = new Object();
```
Comment (wbamberg) [inline live-examples/js-examples/symbol-iterator.html]: "expected output" in this one should be `Array [1, 2, 3]`

File: live-examples/js-examples/weakset-prototype-add.html [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <pre>
-<code id="static-js">var weakset1 = new WeakSet();
-var object1 = {};
+<code id="static-js">const weakset1 = new WeakSet();
+const object1 = {};
```
Comment (wbamberg) [inline live-examples/js-examples/weakset-prototype-add.html]: Very nitpicky, but in Firefox the output is `TypeError: WeakSet value must be an object, got the number 1`

(that is, TypeError not Error)

**Comment by maboa** [comment]
OK thanks for the thorough review. I've updated in line with your comments :)

**Pull request merged**